src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php

```diff
@@ -281,83 +278,19 @@ private function postProcess($eventName)
             }
 
             if (null !== $this->logger && $skipped) {
-                $this->logger->debug('Listener "{listener}" was not called for event "{event}".', array('listener' => $info['pretty'], 'event' => $eventName));
+                $this->logger->debug('Listener "{listener}" was not called for event "{event}".', $context);
             }
 
             if ($listener->stoppedPropagation()) {
                 if (null !== $this->logger) {
-                    $this->logger->debug('Listener "{listener}" stopped propagation of the event "{event}".', array('listener' => $info['pretty'], 'event' => $eventName));
+                    $this->logger->debug('Listener "{listener}" stopped propagation of the event "{event}".', $context);
                 }
 
                 $skipped = true;
             }
         }
     }
 
-    /**
-     * Returns information about the listener.
-     *
-     * @param object $listener  The listener
-     * @param string $eventName The event name
-     *
-     * @return array Information about the listener
-     */
-    private function getListenerInfo($listener, $eventName)
```

Comment:
This method is removed for two reasons:
- computed values are not used anymore in the profiler panel, only the data-clones are (same king of cleanup [already done](https://github.com/symfony/symfony/pull/19614#issuecomment-242171959) in `FormDataExtractor` and in `DebugAccessDecisionManager`);
- to prevent computing the same info several times, it needs to be persistent, and this responsibility belongs to the `WrappedListener` class, that knows best about its wrapped listener.